### ISSUE TITLE
NTR 5-aminolevulinic acid transport

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#19527]
GO:New1:  5-aminolevulinic acid import across plasma membrane
The directed movement of 5-aminolevulinic acid from outside of a cell, across the plasma membrane and into the cytosol.
is_a amino acid import across plasma membrane (GO:0089718) 

GO:New2: 5-aminolevulinic acid transmembrane transporter activity
Enables the transfer of 5-aminolevulinic acid from one side of a membrane to the other.
is_a amino acid transmembrane transporter activity (GO:0015171) 

CHEBI:17549 if useful for xref

PMID:31989647

thanks!

**Comment by ValWood** [comment]
I'm also waiting for this  term.
@ValWood

**Comment by RLovering** [comment]
Hi 
Shirin and I have briefly discussed this ticket. We are surprised that a term is being created for a specific chemical as we have assumed that there will not be a thousands of unique GO terms describing not just the specific chemical transport process (BP term) but also the specific chemical transporter activity (MF term). We had assumed this information would be captured using the general parent terms (even when this is multiple BP/MF terms) with the specific chemical included in the AE.

For example Shirin has created this annotations

P31641 | SLC6A6 | enables | GO:0015171 | amino acid transmembrane transporter activity | ECO:0000314 | IDA | PMID:24842606 |   | 9606 | ARUK-UCL | has_target_start_location(GO:0009897),has_target_end_location(GO:0009898),has_input(CHEBI:17549),part_of(GO:0089718)
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
P31641 | SLC6A6 | involved_in | GO:0089718 | amino acid import across plasma membrane | ECO:0000314 | IDA | PMID:24842606 |   | 9606 | ARUK-UCL | has_target_start_location(GO:0009897),has_target_end_location(GO:0009898),has_input(CHEBI:17549),causally_upstream_of(GO:0010940)
Q9NSD5 | SLC6A13 | enables | GO:0015171 | amino acid transmembrane transporter activity | ECO:0000314 | IDA | PMID:24842606 |   | 9606 | ARUK-UCL | has_target_start_location(GO:0009897),has_target_end_location(GO:0009898),has_input(CHEBI:17549),part_of(GO:0089718)
Q9NSD5 | SLC6A13 | involved_in | GO:0089718 | amino acid import across plasma membrane | ECO:0000314 | IDA | PMID:24842606 |   | 9606 | ARUK-UCL | has_target_start_location(GO:0009897),has_target_end_location(GO:0009898),has_input(CHEBI:17549),causally_upstream_of(GO:0010940)

The other side to this is that would it be possible to check for existing GO annotations that have the ChEBI ID in the AE field and alert curators about the new terms available, eg using the QuickGO URL:
https://www.ebi.ac.uk/QuickGO/annotations?extension=has_input(CHEBI:17549) modified with the appropriate ChEBI ID. Or ideally make these annotation automatically updated in some way.

In summary clearer guidelines are needed to enable curators to know when to use the general terms with AE has_input and when to request new GO terms.

Thanks

Ruth

@krchristie @ShirinSaverimuttu

**Comment by pgaudet** [comment]
I agree. I propose that we only annotate the 'widest' range of substrate that a transporter transports with a GO term, and any specific substrates in extensions. In this case the abstract states "The 18 kDa translocator protein (TSPO) has been proposed to be involved in the transport of porphyrins and heme analogs.", perhaps the definition for heme transmembrane transporter activity should be widened instead.

**Comment by RLovering** [comment]
Thanks for you prompt reply, but what does that mean? Are you saying that you will remove the terms you have created?

**Comment by pgaudet** [comment]
I can if it's OK for @ValWood 

It also means that there is A LOT of work to do to reannotate transporters, and I wanted to finish other stuff, which is why I wasn't too bothered. But better avoid proliferation and avoid more fixing later.

**Comment by pgaudet** [comment]
I would also like for @krchristie to respond , make sure we agree

**Comment by Antonialock** [comment]
I see your point Ruth. 

I also think it will become hard to find GPs annotated to the transport of a specific substrate (only real superusers will end up to this page: https://www.ebi.ac.uk/QuickGO/annotations?extension=has_input(CHEBI:17549)  - is that a developer problem?....

**Comment by ValWood** [comment]
but we have always had MF terms for specific biomolecules that are required?
5-aminolevulinic acid is the first compound in the porphyrin synthesis, and for heme.

I can see that we don't want terms for any  old chemical, but it is pretty important  that we are specific with nutrients.

**Comment by ValWood** [comment]
>I agree. I propose that we only annotate the 'widest' range of substrate that a transporter transports with a GO term, and any specific substrates in extensions. In this case the abstract states "The 18 kDa translocator protein (TSPO) has been proposed to be involved in the transport of porphyrins and heme analogs.", perhaps the definition for heme transmembrane transporter activity should be widened instead.

the pombe people who work on this process are pretty excited about this missing activity. This isn't a heme transporter, it's a heme precursor transporter.


>In summary clearer guidelines are needed to enable curators to know when to use the general terms with AE has_input and when to request new GO terms.

Easy. Terms allowed for  biological molecules  that are part of  nutrient pathways.

**Comment by RLovering** [comment]
great but we need this to be an agreed guideline ;)

**Comment by krchristie** [comment]
@pgaudet - I agree in principle that it would be better to annotate transporters to terms that represent the breadth of their specificity rather than to annotate to lots of individual terms for each of the demonstrated specific substrates. In practice, it may be harder to know whether a given transporter has a broad or narrow specificity when annotating a single paper. 

@Antonialock also brought up a really great point that we don't currently have a way for users to see or use annotation extensions in a good way. That's definitely something I think it would be good for us to think about how we could deal with this better.

In summary, I agree with the discussion above, but I think it will be a constant process to decide what is and isn't appropriate.

**Comment by Antonialock** [comment]
Thanks all for your input. I'll close this ticket. This protein is not a great example to form a discussion over and it's role is quite enigmatic....I'll keep thinking on what I can annotate it to if at all.

**Comment by alanbridge** [comment]
Hi 

sorry to but in but it would be neat if you could xref 5-aminolevulinate / CHEBI:356416 as used in Rhea: 

https://www.rhea-db.org/searchresults?q=CHEBI%3A356416

We don't currently have this paper nor transport reaction for this non-proteogenic amino acid in Rhea, so will ask Rhea curators to check whether we can create such a reaction which could also be xref-ed in the GO MF. 

We plan a large scale retrofit of transport reactions in UniProt next year when we are done with lipid metabolism from SwissLipids and other reference pathways. For transporters our current plan is that we will annotate the specific substrates as we do now for enzymes (i.e. we take the same view as @ValWood). If you annotate to individual terms for each of the demonstrated specific substrates then you can use the ChEBI ontology / chemical structural data to figure out later what the breadth of specificity is and what chemical features define that specificity. People with some background in cheminformatics could take all structures xref-ed in GO terms and extract the common features for a given transporter. If you do high level annotation you lose the possibility to do that (and a lot of other things too, like mapping metabolomics data to proteins via Rhea/GO annotations). Just my 2 cents. 

Pinging @amorgat @hdrabkin for info, hi both!

Have a good day,

All the best, Alan

**Comment by Antonialock** [comment]
@alanbridge 

The 'problem' with this protein (TSPO2) is that there are a handful of papers that seem to follow at least 2 quite different themes as to what this protein does. 

In PMID:31989647 (the one referenced for this NTR) they show that when you overexpress the protein, activity goes up, and when you downregulate it, the activity goes down. Adding to that the protein name is short for 'translocator protein' -seemed like a very straight forward protein to hang some annotation on to. (in this paper they use cancer cells but in the introduction the authors state that these cell lines are used as 'models' for different stages of erythrocyte differentiation)

In support of TSPO2 having some role in transport, it's also been shown to be in a 'supramolecular spectrin associated membrane complex' where it provides a 'secondary' pathway (in addition to a well known mechanism) to promote ATP efflux from erythrocytes. The evidence for a physical complex is not completely convincing, and most (?) of the evidence is a little bit indirect, but the authors (and reviewers....) are convinced so maybe I just don't know enough about the assays and systems. e.g. 27641616 & 30061676 (the second shows a model fig 8 for how they think this happens)

In addition to the transport role, there is also evidence in dog, human and mouse (19729679, 32358067) that it is involved in lipid distribution - it's been shown to bind cholesterol, and in vitro assays implicate it in lipid droplet metabolism, which is in line with what's been shown for plants/yeast (30003614).

**Comment by ValWood** [comment]
I'll remove my ISS then! That said, the fission yeast "iron accumulation" expert is quite excited about the possibility and testing, so hopefully we will know more soon. 
There is a strong connection between iron homeostasis and lipid storage though...

**Comment by ValWood** [comment]
>We plan a large scale retrofit of transport reactions in UniProt next year when we are done with lipid metabolism from SwissLipids and other reference pathways.

Looking forward to this!

**Comment by amorgat** [comment]
Transport reaction for CHEBI:356416
RHEA:64816 5-aminolevulinate(in) = 5-aminolevulinate(out) 
It will be publicly available in the Rhea release 115 (Oct 7 2020)

**Comment by pgaudet** [comment]
@Antonialock 
Do you want those terms, or not ? 

Since they are in Rhea, maybe it's OK to create them ? @cmungall what do you think ?

**Comment by ValWood** [comment]
I think that in general we should allow 'nutrient' transporters, but  we would not want a specific transporter term for every xenobiotic. Does that sound sensible?
Eveything that is actively imported and has a metabolic role, but not  xenobiotic exporters which can be more general, we could use CHEBI ID in substrate.

Pending decision, at the moment for this one  we are waiting for better data to annotate, but will need it later, so you can close for now or add.

**Comment by pgaudet** [comment]
I'll add the terms, since that was already done in that branch. 

We can decide later exactly which terms are allowed. I thought we should add it because there is a Rhea reaction, although I dont think we want (or can) to be aligned with Rhea 100%. 

Thanks, Pascale

**Comment by deustp01** [comment]
> I dont think we want (or can) to be aligned with Rhea 100%.

@ukemi ? @hdrabkin For MF, aren't we already committed to complete alignment?

**Comment by pgaudet** [comment]
Well for one thing we dont do the reciprocal reactions, do we ?

**Comment by alanbridge** [comment]
Hi 

just for info @lbreuza is gearing up to do a retrofit of transporters in UniProt at the moment. Given the huge number of drugs that could be transported we decided to focus on compounds that the transporter (likely) evolved to transport. So you would annotate "nutrients" as mentioned by Val, and also toxic compounds like heavy metals too (the transporters involved in their elimination), but not necessarily all synthetic drugs that recently came on the scene.  

All the best, Alan

**Comment by pgaudet** [comment]
That's great. @lbreuza we should try to coordinate on this.

**Comment by pgaudet** [comment]
So, WRT TSPO (P30536 in human) - is 5-aminolevulinic acid specific, or should I obsolete those terms ??

**Comment by pgaudet** [comment]
I'll close, please reopen if this was not right.

**Comment by pgaudet** [comment]
Removed incorrect compound in LD c-GMP-AMP(2-)
